Drv/Ip/TcpServerSocket.cpp

```diff
@@ -84,8 +84,8 @@ SocketIpStatus TcpServerSocket::startup() {
     }
     U16 port = ntohs(address.sin_port);
     Fw::Logger::logMsg("Listening for single client at %s:%hu\n", reinterpret_cast<POINTER_CAST>(m_hostname), port);
-    // TCP requires listening on the socket. Second argument prevents queueing of anything more than a single client.
-    if (::listen(serverFd, 0) < 0) {
+    // TCP requires listening on the socket. Second argument sets backlog to 1 since TCP is expecting a client.
```

Comment:
What about phrasing the comment as "Since we only expect a single client, set the TCP backlog (the second argument) to 1 to prevent queuing of multiple clients."

Setting the backlog to 1 would allow a single client to queue after the initial connection, but I think that's the best that can be done given the architecture of linux TCP, and is definitely less confusing that just dropping all connections on a subset of linux systems.